src/main/java/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
+import onlinebookstore.validation.Isbn;
 
 public record CreateBookRequestDto(
-        Long id, String title, String author, String isbn,
-        BigDecimal price, String description, String coverImage) {
+        Long id,
+        @NotNull @Size(min = 2) String title,
+        @NotNull @Size(min = 2) String author,
```

Comment:
why size 2?

Reply:
I just decided it should be posible to create book with title and author with size 2 and more. But better decision here will be to use `@NotBlank` annotation